AdminOpsGuide/dbrepl.rst

```diff
@@ -1191,141 +1193,312 @@ Here is the code:
+   %GDE-I-LOADGD, Loading Global Directory file
+        /home/A/yottadb.gld
+   %GDE-I-VERIFY, Verification OK
+
+   %GDE-I-EXECOM, Executing command file /home/A/gdemsr
+   %GDE-I-VERIFY, Verification OK
+
+   %GDE-I-GDUPDATE, Updating Global Directory file
+        /home/A/yottadb.gld
+   %YDB-I-DBFILECREATED, Database file /home/A/yottadb.dat created
+
+.. parsed-literal::
+   $./repl_setup
+   %YDB-I-JNLCREATE, Journal file /home/A/yottadb.mjl created for region DEFAULT with BEFORE_IMAGES
+   %YDB-I-JNLSTATE, Journaling state for region DEFAULT is now ON
+   %YDB-I-REPLSTATE, Replication state for region DEFAULT is now ON
 
-* Turn on replication.
-* Create the replication instance file.
 * Start the Source Server.
 
+.. parsed-literal::
+   ./originating_start A B 4001
+   Wed Jul 18 12:45:46 2018 : Initiating START of source server for secondary instance [B]
+   Wed Jul 18 12:45:46 2018 : Initiating CHECKHEALTH operation on source server pid [18498] for secondary instance name [B]
+   PID 18498 Source server is alive in ACTIVE mode
+   Wed Jul 18 12:45:46 2018 : /usr/local/lib/yottadb/r122/mupip replicate -source -start -instsecondary=B -secondary=localhost:4001 -buffsize=1048576 -log=/home/A/A_B.log
+   Wed Jul 18 12:45:46 2018 : %YDB-I-REPLINFO, GTM Replication Source Server with Pid [18498] started for Secondary Instance [B]
+   Wed Jul 18 12:45:46 2018 : Created jnlpool with shmid = [1353941014] and semid = [1447755780]
+   Wed Jul 18 12:45:46 2018 : %YDB-I-REPLINFO, GTM Replication Source Server now in ACTIVE mode using port 4001
+   Wed Jul 18 12:45:46 2018 : Connect hard tries count = 5, Connect hard tries period = 500
+   Wed Jul 18 12:45:46 2018 : 1 hard connection attempt failed : Connection refused
+
+Once B is brought up as replicating, check the log created. For example,
+
+.. parsed-literal::
+   $ tail -f A/A_B.log
+   Wed Jul 18 12:46:04 2018 : Waiting for REPL_INSTINFO message
+   Wed Jul 18 12:46:04 2018 : Received REPL_INSTINFO message
+   Wed Jul 18 12:46:04 2018 : Received secondary instance name is [B]
+   Wed Jul 18 12:46:04 2018 : Current Journal Seqno of the instance is 1 [0x1]
+   Wed Jul 18 12:46:04 2018 : Source server last sent seqno 1 [0x1]
+   Wed Jul 18 12:46:04 2018 : Source server will start sending from seqno 1 [0x1]
+   Wed Jul 18 12:46:04 2018 : Source server now reading from the journal POOL
+   Wed Jul 18 12:46:04 2018 : Sending REPL_WILL_RESTART_WITH_INFO message with seqno 1 [0x1]
+   Wed Jul 18 12:46:04 2018 : Sending REPL_HISTREC message with seqno 1 [0x1]
+   Wed Jul 18 12:46:04 2018 : New History Content : Start Seqno = 1 [0x1] : Stream Seqno = 0 [0x0] : Root Primary = [A] : Cycle = [1] : Creator pid = 18498 : Created time = 1531932346 [0x5b4f6eba] : History number = 0 : Prev History number = -1 : Stream # = 0 : History type = 1
+
 On B:
 
-* Turn on replication.
-* Create a new replication instance file.
+* Source the environment file.
+* Create the database and the replication instance file.
```

Comment:
This line is a duplicate of line 1263. Similarly, Lines 1257 and 1258 are duplicates of 1269 and 1270.

Reply:
They were duplicates even earlier, with the lines you see previously, which don't explain about sourcing the environment file. Basically, the same steps are done on both A and B, and it helps to see that the steps are identical.

Reply:
Thanks for the offline clarification - I see the duplication and will fix it.